pom.xml

```diff
@@ -12,6 +12,7 @@
     <packaging>hpi</packaging>
     <properties>
         <jenkins.version>2.7.3</jenkins.version>
+        <jenkinsRuleTimeout>-Djenkins.test.timeout=480</jenkinsRuleTimeout>
```

Comment:
Just move the timeout value to the property, the compiler flag should still live in the argLine tag.

Reply:
Updated